src/views/OrderDetail.vue

```diff
@@ -17,6 +17,14 @@
             <ion-badge v-if="order.placedDate" color="dark" slot="end">{{ timeFromNow(order.placedDate) }}</ion-badge>
           </ion-item>
         </ion-list>
+        <ion-item v-if="order.shippingInstructions" color="light" lines="none">
+          <ion-label class="ion-text-wrap">
+            <h3>{{ $t("Handling Instructions") }}</h3>
+            <ion-text color="medium">
```

Comment:
we can use p tag directly here, don't need to use ion text for color. if UI is off from figma a little, that's fine.

Reply:
Okay sir

Reply:
@dt2patel  Sir here thing is that if I am using 
color="light" property of ion-item for background color
then the p tag does not inherit the gray color.

And looks something like this:- 

![3333](https://github.com/hotwax/bopis/assets/67117461/dc0deb17-4632-4438-9579-22f44d2a972a)